src/plat/bcm2837/overlay-rpi3.dts

```diff
@@ -20,4 +20,14 @@
 		/* This is configurable in the Pi's config.txt, but we use 128MiB of RAM by default. */
 		reg = <0x00 0x08000000>;
 	};
+
+	reserved-memory {
```

Comment:
Please add a comment here explaining in detail why this memory is reserved.

Reply:
Fixed!